riscv-debug-spec.tex

```diff
@@ -348,13 +412,13 @@ \section{Debug Module (DM)} \label{dm}
     operations:}
 \item Provide access to a reset signal that allows debugging out of reset.
     (Required)
-\item Allow any individual hart to be halted. (Required)
+\item Allow any individual hart to be halted and resumed. (Required)
 \item Provide status on which harts are halted. (Required)
-\item Provide access to a halted hart's GPRs. (Required)
-\item Give the debugger necessary information about the implementation.
-    (Required)
+\item Provide read (and write?) access to a halted hart's GPRs. (Required)
+\item Give the debugger necessary information about the implementation. (Required)
 \item Provide access to other hart registers. (Optional)
 \item Force the hart to execute arbitrary instructions. (Optional)
+\item Single step? Is that required? What about seeing / modifying PC?
```

Comment:
Ok, I guess i was wondering if having *some* mechanism to read and/or write the PC is optional. Sounds like it is, though that seems like it would make debugging pretty hard.